raspberry/bzzz/scheduler.py

```diff
@@ -0,0 +1,95 @@
+import bzzz.thread_this
+from time import time_ns
+
+
+class Scheduler:
+    # Dict keys
+    _function_ID = "function_ID"
+    _function_obj = "function_obj"
+    _function_call_frequency = "function_call_frequency"
+    _function_call_count = "function_call_count"
+    _function_thread = "function_thread"
+    _num_times_called = "num_times_called"
+    _time_at_last_call_ns = "time_at_last_call_ns"
+    _dummy_func = "dummy_func"
+
+    def __init__(self, use_threading=False) -> None:
```

Comment:
Constructors do not return `None`. Make sure you correct this throughout the entire code.